content/community/contributing/handbook/back-end/nuget-package/_index.md

```diff
@@ -139,10 +139,10 @@ package generation and publishing.
             with:
               fetch-depth: 0
 
-          - name: Install dotnet6
+          - name: Install dotnet8
```

Comment:
If we don't specify the version we don't need to update the name with each upgrade
```suggestion
          - name: Install dotnet
```